src/components/BookForm.js

```diff
@@ -44,18 +45,20 @@ function BookForm() {
           value={author}
           onChange={onAuthorChange}
         />
-        <select
-          className="form-select"
-          id="select-btn"
-          value={category}
-          onChange={(e) => setCategory(e.target.value)}
-        >
-          <option value="empty">select category</option>
-          <option value="Action">Action</option>
-          <option value="Science fiction">Science fiction</option>
-          <option value="Economy">Economy</option>
-        </select>
-        <Button type="submit" onClick={saveBook} btnName="Add-btn" btnValue="Add Book" />
+        <div className="form-last">
+          <select
+            className="form-select"
+            id="select-btn"
+            value={category}
+            onChange={(e) => setCategory(e.target.value)}
+          >
+            <option value="empty">Category</option>
+            <option value="Action">Action</option>
+            <option value="Science fiction">Science fiction</option>
+            <option value="Economy">Economy</option>
+          </select>
+          <Button type="submit" onClick={saveBook} btnName="Add-btn" btnValue="Add Book" />
+        </div>
```

Comment:
- [ ] It would be nice if you could style the form elements to span through left and right and align with the book cards so spacing and alignment are achieved as in the original design. Please take a look at the screenshot and compare the original design to yours.  Kindly work on that 👍🏽